serviceconnector/src/tree/ServiceConnectorGroupItem.ts

```diff
@@ -22,11 +22,11 @@ export class ServiceConnectorGroupItem implements TreeElementBase {
     async getChildren(): Promise<ServiceConnectorItem[]> {
         const result = await callWithTelemetryAndErrorHandling('getChildren', async () => {
             const client = new (await import('@azure/arm-servicelinker')).ServiceLinkerManagementClient(this.subscription.credentials);
-            const linkers = (await uiUtils.listAllIterator(client.linker.list(nonNullValue(this.item.id))));
+            const linkers = (await uiUtils.listAllIterator(client.linker.list(nonNullProp(this.item, 'id'))));
             return linkers.map(linker => createServiceConnectorItem(this.subscription, this.item, linker));
         });
 
-        return nonNullValue(result);
+        return nonNullValue(result, 'getChildren');
```

Comment:
I don't understand this change here.

Reply:
I added 'getChildren' as a `propertyNameOrMessage` since I still wanted to have some context as to what value is nonNull. I couldn't change this to nonNullProp since result isn't a property. Not sure if this is actually what this optional parameter does since it doesn't give a description.